backend/tests/test_app.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from flask import json
+import server.app as ap
+
+
+@pytest.fixture(scope='module')
+def response_get():
```

Comment:
Using a fixture with a context manager here is an overkill. 

You don't need to manage resources like connections or memory in this case. There is no need to free a resource after you processed the response (resp). Therefore it could be just a normal function that you call from two tests. 

For example:
```
def response_get():
   with ap.app.test_client() as con:
        return con.get('/')
```

Maybe later we will need to simplify that a bit.

@cornhubius @leopardilla what do you think? Is it ok to leave it for now?

But it's great that you understood how it works :+1:

Reply:
I think it's ok for now (I've found some info about fixtures in pytest)
If @DramaTourG likes this solution we can leave it :)

Reply:
maybe we should remove the fixture, I see a ghoul - yield😂